lang/id.po

```diff
@@ -0,0 +1,99 @@
+msgid ""
+msgstr ""
+"Language: id\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+# A place for you to credit yourself!
+# Put something like "Latin translation by Jvlivs Gaivs\n" here.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "Diterjemahkan ke Bahasa oleh Habib Fikri\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d paket free (dari total %s)\n"
+msgstr[1] "%d paket free (dari total %s)\n"
+
+msgid "NONFREE_PACKAGES_COUNT\n"
+msgid_plural "NONFREE_PACKAGES_COUNT\n"
+msgstr[0] "%d paket non-free (dari total %s)\n"
+msgstr[1] "%d paket non-free (dari total %s)\n"
+
+msgid "RMS_HAPPY\n"
+msgstr "Hanya paket free - rms akan bangga!\n"
+
+msgid "RMS_DISAPPOINTED\n"
+msgstr "Lebih dari 10%% paket non-free. Apakah Anda benci kebebasan?\n"
+
+msgid "HELP_USAGE\n"
+msgstr "Penggunaan: vrms-rpm [opsi]\n"
+
+msgid "HELP_OPTION_ASCII\n"
+msgstr "    Tampilkan rms ASCII-art ketika tidak ditemukan paket non-free,\n"
+       "    atau ketika terdapat paket non-free lebih dari 10%% dari total.\n"
+
+msgid "HELP_OPTION_COLOUR\n"
+msgstr "    Mengontrol apakah terminal escape sequences dipergunakan\n"
+       "    untuk mewarnai output. Nilai defaultnya 'auto', yang mana menggunakan output berwarna\n"
+       "    ketika menulis di terminal, tetapi tidak berlaku apabila menulis di file atau pipe.\n"
```

Comment:
Each line of the translated text should fit within 80 characters. You don't have to fit the text in 3 lines, though - you can split it into 4 lines or more, if needed.

Reply:
Sorry, I didn't make it clear enough - this is 80 characters from the opening `"` to the closing `"`, i.e. including the trailing 4-space indent.

Reply:
Does this only apply to `HELP_OPTION`?

Reply:
Yes. The `ERR_` messages should **not** be wrapped.